reactor-netty-http/src/main/java/reactor/netty/http/server/ConnectionInfo.java

```diff
@@ -94,12 +96,18 @@ static ConnectionInfo from(Channel channel, HttpRequest request, boolean secured
 
 	ConnectionInfo(SocketAddress hostAddress, String hostName, int hostPort,
 			SocketAddress remoteAddress, String scheme, boolean isInetAddress) {
+		this(hostAddress, hostName, hostPort, remoteAddress, scheme, isInetAddress, "");
+	}
+
+	ConnectionInfo(SocketAddress hostAddress, String hostName, int hostPort,
```

Comment:
I added a new constructor, but I can absolutely go ahead and modify all the usages of the above constructor to add the last parameter and remove the above.

Reply:
I like it with a new constructor